scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -48,6 +48,12 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
                                 </div>
                             </div>
                             <div class="field">
+                                <label translate>Desk language</label>
+                                <input type="text" class="line-input fullwidth-input"
```

Comment:
imo there should be a select box with some predefined values, we need those to match with dictionary language code, and also with language you can set in authoring, and this would be useful everywhere